src/plugins/daoModule/index.ts

```diff
@@ -312,4 +374,38 @@ export default class Plugin {
     const receipt = await tx.wait();
     console.log('[DAO module] executed proposal:', receipt);
   }
+
+  async voteForQuestion(
+    web3: any,
+    oracleAddress: string,
+    questionId: string,
+    minimumBond: string,
+    answer: '1' | '0'
+  ) {
+    const currentBond = await call(web3, OracleAbi, [
+      oracleAddress,
+      'getBond',
+      [questionId]
+    ]);
+    const bond = currentBond.eq(BigNumber.from(0))
+      ? BigNumber.from(minimumBond)
+      : currentBond.mul(2);
+
+    const tx = await sendTransaction(
+      web3,
+      oracleAddress,
+      OracleAbi,
+      'submitAnswer',
+      [
+        questionId,
+        `0x000000000000000000000000000000000000000000000000000000000000000${answer}`,
+        bond
+      ],
+      {
+        value: bond.toString()
+      }
+    );
+    const receipt = await tx.wait();
+    console.log('[DAO module] executed vote on oracle:', receipt);
```

Comment:
Here you could return the tx directly so it's possible to get it on the UI and do thing like tx.wait(1) instead of sleep().

Reply:
Is this a pattern that we want to apply? What you are saying makes total sense but I just wanted to keep the same pattern used in other `sendTransaction` methods @bonustrack

Reply:
Sorry didn't noticed others functions use the same pattern, this is just a suggestion, you can keep it the way it work, it can be changed later for all function if that become necessary.